docs/resources/artifactory_remote_vcs_repository.md

```diff
@@ -0,0 +1,36 @@
+# Artifactory Remote Go Repository Resource
+
+Creates a remote VCS repository.
+Official documentation can be found [here](https://www.jfrog.com/confluence/display/JFROG/VCS+Repositories)
+
+
+## Example Usage
+To create a new Artifactory remote VCS repository called my-remote-vcs.
+
+```hcl
+resource "artifactory_remote_go_repository" "my-remote-vcs" {
+  key                         = "my-remote-vcs"
+  url                         = "https://github.com/"
+  vcs_git_provider            = "GITHUB"
+  max_unique_snapshots        = 5
+}
+```
+
+## Argument Reference
+
+Arguments have a one to one mapping with the
+[JFrog API](https://www.jfrog.com/confluence/display/JFROG/Repository+Configuration+JSON#RepositoryConfigurationJSON-RemoteRepository).
+The following arguments are supported:
+
+* `key` - (Required) The repository identifier. Must be unique system-wide
+* `description` - (Optional)
+* `notes` - (Optional)
+* `url` - (Required) - the remote repo URL.
+* `vcs_git_provider` - (Optional) The maximum number of unique snapshots of a single artifact to store. Once the 
```

Comment:
Looks like the description for 'vcs_git_provider' is incorrect. It is copy of 'max_unique_snapshots'.

Reply:
Also, when 'vcs_git_provider' value is "CUSTOM" ... we need to specify one more field ie: 'vcs_git_download_url' 
We already have a vcs specific schema ... Please check composer and cocoapods remote repo resources for reference.

Reply:
Explain? I think [`vcsRemoteRepoSchema`](https://github.com/jfrog/terraform-provider-artifactory/blob/master/pkg/artifactory/repositories.go#L756) is the correct schema to use, per @maheshjfrog comment.

Reply:
description is fixed

Reply:
@maheshjfrog you're right. Use your schema now.